CIP-0005/README.md

```diff
@@ -121,22 +118,25 @@ We define the following set of common prefixes with their corresponding semantic
 | `addr_test`         | Testnet address                                       | Network tag, payment credential and optional stake credential |
 | `stake`             | Mainnet stake address                                 | Network tag and stake credential                              |
 | `stake_test`        | Testnet stake address                                 | Network tag and stake credential                              |
-| `drep`              | drep identifier                                       | drep credential, see [CIP-0129]                               |
-| `cc_cold`           | cc cold identifier                                    | cc cold credential, see [CIP-0129]                            |
-| `cc_hot`            | cc hot identifier                                     | cc hot credential, see [CIP-0129]                             |
-| `gov_action`        | gov action identifier                                 | gov action ID, see [CIP-0129],                                |
+| `drep`              | DRep identifier                                       | DRep credential, see [CIP-0129]                               |
+| `cc_cold`           | Constitutional committee cold identifier              | Constitutional committee cold credential, see [CIP-0129]      |
+| `cc_hot`            | Constitutional committee hot identifier               | Constitutional committee hot credential, see [CIP-0129]       |
+| `gov_action`        | Governance action identifier                          | Governance action ID, see [CIP-0129]                          |
 
 
 ### Deprecated Governance Prefixes
-The prefixes above are the version defined by [CIP-0129] and should be used at this time. The prefixes below were previously defined by CIP-0105, and are deprecated. Please see CIP-0105 for the detailed deprecation information, and details to upgrade to [CIP-0129].
+
+The prefixes above are the version defined by [CIP-0129] and should be used at this time. The prefixes below were previously defined by [CIP-0105], and are deprecated. Please see [CIP-0105] for the detailed deprecation information, and details to upgrade to [CIP-0129].
```

Comment:
Aren't we already in CIP-105? This thing is linking is telling people to read itself for the detailed deprecation information? Perhaps if that is going to be a link it can at least link to the section that discussions said deprecation?

Reply:
ill improve the wording to avoid ambiguity

Reply:
This should improve it
[d22f35f](https://github.com/cardano-foundation/CIPs/pull/965/commits/d22f35fa0b61d74df065f2e4917e9437d5baa71c)

Reply:
Looks like the commit above fixes it: resolving.